src/main/java/edu/kis/powp/jobs2d/command/manager/DriverManagerObserver.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverManagerObserver implements Subscriber {
```

Comment:
The name should indicate what is done in update method.  If not, every observer would have the same name!